frontend/src/components/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -111,7 +113,7 @@ const RegistrationsCard = styled(Card)({
 type PlayersListProps = { players: PlayerProfileDto[] };
 const PlayersList = (props: PlayersListProps) => {
   if (props.players.length === 0) {
-    return <PlayerNotFound />;
+    return PlayerNotFound();
```

Comment:
Co tutaj się stało?

Reply:
to jest dobre pytanie... coś grzebałem przy komponencie i dopiero jak tak zmieniłem to mi łapało dobrze :-p